actor/src/main/scala/org/apache/pekko/actor/Actor.scala

```diff
@@ -100,8 +100,8 @@ final case class ActorIdentity(correlationId: Any, ref: Option[ActorRef]) {
    * not defined if no actor matched the request.
    */
   def getActorRef: Optional[ActorRef] = {
-    import scala.compat.java8.OptionConverters._
-    ref.asJava
+    import pekko.util.OptionConverters._
+    ref.toJava
```

Comment:
so all the `as` changed to `to`?

Reply:
And I think the title should be `OptionConverters `?

Reply:
> And I think the title should be `OptionConverters `?

Don't understand what you mean by this? Is there a typo or are you thinking of something else.

Reply:
@mdedetrich I was trying to find a class named `ObjectConverters`

Reply:
Ah so you can find it when reviewing?

Reply:
My bad, a had a mental blip. `ObjectConverters` is meant to be `OptionConverters`. Edited original post to also reflect this.